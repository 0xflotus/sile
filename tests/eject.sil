\begin[papersize=a5]{document}
\script[src=packages/lorem]
\pagetemplate[first-content-frame=a]{
\frame[id=a,top=5%,bottom=95%,next=b,left=5%,right=28%]
\frame[id=b,top=5%,bottom=95%,next=c,left=33%,right=62%]
\frame[id=c,top=5%,bottom=95%,left=66%,right=95%]
}

1 \lorem[words=60]

2 \lorem[words=60]

\framebreak

3 \lorem[words=60]

\pagebreak

4 \lorem[words=60]

\end{document}
