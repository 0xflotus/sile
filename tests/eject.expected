<tests/eject.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.97638
My 	37.33214
Set font 	Gentium;10;200;regular;normal;;LTR
T	17	(1)
Mx 	48.55194
T	76 79 82 69 77	(lorem)
Mx 	75.78571
T	73 80 83 85 77	(ipsum)
Mx 	103.86908
T	68 79	(do)
Mx 	114.09858
T	13	(-)
Mx 	20.97638
My 	51.33214
T	76 79 82	(lor)
Mx 	35.98684
T	83 73 84	(sit)
Mx 	49.31273
T	65 77 69 84	(amet)
Mx 	73.30756
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	20.97638
My 	65.33214
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	65.88295
T	69 76 73 84 82	(elitr)
Mx 	85.23289
T	83 69 68	(sed)
Mx 	100.82306
T	68 73 65 77	(diam)
Mx 	20.97638
My 	79.33214
T	78 79 78 85 77 89	(nonumy)
Mx 	57.21108
T	69 73 82 77 79 68	(eirmod)
Mx 	88.67039
T	84 69 77 80 79 82	(tempor)
Mx 	20.97638
My 	93.33214
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	58.21206
T	85 84	(ut)
Mx 	68.86082
T	76 65 66 79 82 69	(labore)
Mx 	96.75568
T	69 84	(et)
Mx 	106.72573
T	68 79	(do)
Mx 	116.95522
T	13	(-)
Mx 	20.97638
My 	107.33214
T	76 79 82 69	(lore)
Mx 	39.20327
T	77 65 71 78 65	(magna)
Mx 	68.71922
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	108.64044
T	69 82 65 84	(erat)
Mx 	20.97638
My 	121.33214
T	83 69 68	(sed)
Mx 	36.56655
T	68 73 65 77	(diam)
Mx 	59.00730
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	97.51251
T	65 84	(at)
Mx 	107.45327
T	86 69 82 79	(vero)
Mx 	20.97638
My 	135.33214
T	69 79 83	(eos)
Mx 	37.36850
T	69 84	(et)
Mx 	48.31140
T	65 67 67 85 83 65 77	(accusam)
Mx 	86.30509
T	69 84	(et)
Mx 	97.24799
T	74 85 83 84 79	(justo)
Mx 	20.97638
My 	149.33214
T	68 85 79	(duo)
Mx 	38.41225
T	68 79 76 79 82 69 83	(dolores)
Mx 	70.73094
T	69 84	(et)
Mx 	80.70099
T	69 65	(ea)
Mx 	91.81850
T	82 69 66 85 77	(rebum)
Mx 	20.97638
My 	163.33214
T	83 84 69 84	(stet)
Mx 	38.25112
T	67 76 73 84 65	(clita)
Mx 	57.98191
T	75 65 83 68	(kasd)
Mx 	78.54278
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	20.97638
My 	177.33214
T	78 79	(no)
Mx 	33.43178
T	83 69 65	(sea)
Mx 	48.41160
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	86.71661
T	83 65 78 67 84 85 83	(sanctus)
Mx 	20.97638
My 	191.33214
T	69 83 84	(est)
Mx 	37.26288
T	76 79 82 69 77	(lorem)
Mx 	65.97614
T	73 80 83 85 77	(ipsum)
Mx 	95.53901
T	68 79 76 79 82	(dolor)
Mx 	20.97638
My 	205.33214
T	83 73 84	(sit)
Mx 	32.89956
T	65 77 69 84	(amet)
Mx 	55.49168
T	76 79 82 69 77	(lorem)
Mx 	81.75079
T	73 80 83 85 77	(ipsum)
Mx 	108.85952
T	68 79	(do)
Mx 	119.08901
T	13	(-)
Mx 	20.97638
My 	219.33214
T	76 79 82	(lor)
Mx 	35.98684
T	83 73 84	(sit)
Mx 	49.31273
T	65 77 69 84	(amet)
Mx 	73.30756
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	20.97638
My 	233.33214
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	65.88295
T	69 76 73 84 82	(elitr)
Mx 	85.23289
T	83 69 68	(sed)
Mx 	100.82306
T	68 73 65 77	(diam)
Mx 	40.97638
My 	248.33214
T	18	(2)
Mx 	48.55194
T	76 79 82 69 77	(lorem)
Mx 	75.78571
T	73 80 83 85 77	(ipsum)
Mx 	103.86908
T	68 79	(do)
Mx 	114.09858
T	13	(-)
Mx 	20.97638
My 	262.33214
T	76 79 82	(lor)
Mx 	35.98684
T	83 73 84	(sit)
Mx 	49.31273
T	65 77 69 84	(amet)
Mx 	73.30756
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	20.97638
My 	276.33214
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	65.88295
T	69 76 73 84 82	(elitr)
Mx 	85.23289
T	83 69 68	(sed)
Mx 	100.82306
T	68 73 65 77	(diam)
Mx 	20.97638
My 	290.33214
T	78 79 78 85 77 89	(nonumy)
Mx 	57.21108
T	69 73 82 77 79 68	(eirmod)
Mx 	88.67039
T	84 69 77 80 79 82	(tempor)
Mx 	20.97638
My 	304.33214
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	58.21206
T	85 84	(ut)
Mx 	68.86082
T	76 65 66 79 82 69	(labore)
Mx 	96.75568
T	69 84	(et)
Mx 	106.72573
T	68 79	(do)
Mx 	116.95522
T	13	(-)
Mx 	20.97638
My 	318.33214
T	76 79 82 69	(lore)
Mx 	39.20327
T	77 65 71 78 65	(magna)
Mx 	68.71922
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	108.64044
T	69 82 65 84	(erat)
Mx 	20.97638
My 	332.33214
T	83 69 68	(sed)
Mx 	36.56655
T	68 73 65 77	(diam)
Mx 	59.00730
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	97.51251
T	65 84	(at)
Mx 	107.45327
T	86 69 82 79	(vero)
Mx 	20.97638
My 	346.33214
T	69 79 83	(eos)
Mx 	37.36850
T	69 84	(et)
Mx 	48.31140
T	65 67 67 85 83 65 77	(accusam)
Mx 	86.30509
T	69 84	(et)
Mx 	97.24799
T	74 85 83 84 79	(justo)
Mx 	20.97638
My 	360.33214
T	68 85 79	(duo)
Mx 	38.41225
T	68 79 76 79 82 69 83	(dolores)
Mx 	70.73094
T	69 84	(et)
Mx 	80.70099
T	69 65	(ea)
Mx 	91.81850
T	82 69 66 85 77	(rebum)
Mx 	20.97638
My 	374.33214
T	83 84 69 84	(stet)
Mx 	38.25112
T	67 76 73 84 65	(clita)
Mx 	57.98191
T	75 65 83 68	(kasd)
Mx 	78.54278
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	20.97638
My 	388.33214
T	78 79	(no)
Mx 	33.43178
T	83 69 65	(sea)
Mx 	48.41160
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	86.71661
T	83 65 78 67 84 85 83	(sanctus)
Mx 	20.97638
My 	402.33214
T	69 83 84	(est)
Mx 	37.26288
T	76 79 82 69 77	(lorem)
Mx 	65.97614
T	73 80 83 85 77	(ipsum)
Mx 	95.53901
T	68 79 76 79 82	(dolor)
Mx 	20.97638
My 	416.33214
T	83 73 84	(sit)
Mx 	32.89956
T	65 77 69 84	(amet)
Mx 	55.49168
T	76 79 82 69 77	(lorem)
Mx 	81.75079
T	73 80 83 85 77	(ipsum)
Mx 	108.85952
T	68 79	(do)
Mx 	119.08901
T	13	(-)
Mx 	20.97638
My 	430.33214
T	76 79 82	(lor)
Mx 	35.98684
T	83 73 84	(sit)
Mx 	49.31273
T	65 77 69 84	(amet)
Mx 	73.30756
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	20.97638
My 	444.33214
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	65.88295
T	69 76 73 84 82	(elitr)
Mx 	85.23289
T	83 69 68	(sed)
Mx 	100.82306
T	68 73 65 77	(diam)
Mx 	158.44410
My 	37.33214
T	19	(3)
Mx 	167.00560
T	76 79 82 69 77	(lorem)
Mx 	195.22530
T	73 80 83 85 77	(ipsum)
Mx 	224.29461
T	68 79 76 79 82	(dolor)
Mx 	250.09244
T	83 73 84	(sit)
Mx 	138.44410
My 	51.33214
T	65 77 69 84	(amet)
Mx 	161.03622
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	207.10490
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	252.01148
T	69 76 73 84 82	(elitr)
Mx 	138.44410
My 	65.33214
T	83 69 68	(sed)
Mx 	154.03427
T	68 73 65 77	(diam)
Mx 	176.47502
T	78 79 78 85 77 89	(nonumy)
Mx 	212.70972
T	69 73 82 77 79 68	(eirmod)
Mx 	244.16903
T	84 69 77	(tem)
Mx 	260.26278
T	13	(-)
Mx 	138.44410
My 	79.33214
T	80 79 82	(por)
Mx 	155.86361
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	194.32316
T	85 84	(ut)
Mx 	206.19580
T	76 65 66 79 82 69	(labore)
Mx 	235.31453
T	69 84	(et)
Mx 	246.50846
T	68 79	(do)
Mx 	256.73795
T	13	(-)
Mx 	138.44410
My 	93.33214
T	76 79 82 69	(lore)
Mx 	157.12034
T	77 65 71 78 65	(magna)
Mx 	187.08565
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	227.45623
T	69 82 65 84	(erat)
Mx 	246.42545
T	83 69 68	(sed)
Mx 	138.44410
My 	107.33214
T	68 73 65 77	(diam)
Mx 	162.25423
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	202.12882
T	65 84	(at)
Mx 	213.43896
T	86 69 82 79	(vero)
Mx 	235.25691
T	69 79 83	(eos)
Mx 	252.04557
T	69 84	(et)
Mx 	138.44410
My 	121.33214
T	65 67 67 85 83 65 77	(accusam)
Mx 	176.63939
T	69 84	(et)
Mx 	187.78389
T	74 85 83 84 79	(justo)
Mx 	211.08660
T	68 85 79	(duo)
Mx 	229.69693
T	68 79 76 79 82 69 83	(dolores)
Mx 	138.44410
My 	135.33214
T	69 84	(et)
Mx 	148.50886
T	69 65	(ea)
Mx 	159.72108
T	82 69 66 85 77	(rebum)
Mx 	188.71163
T	83 84 69 84	(stet)
Mx 	206.08108
T	67 76 73 84 65	(clita)
Mx 	225.90658
T	75 65 83 68	(kasd)
Mx 	246.56217
T	71 85	(gu)
Mx 	256.73795
T	13	(-)
Mx 	138.44410
My 	149.33214
T	66 69 82	(ber)
Mx 	152.10132
T	71 82 69 78	(gren)
Mx 	172.98446
T	78 79	(no)
Mx 	185.43986
T	83 69 65	(sea)
Mx 	200.41968
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	238.72469
T	83 65 78 67	(sanc)
Mx 	257.06454
T	13	(-)
Mx 	138.44410
My 	163.33214
T	84 85 83	(tus)
Mx 	154.17513
T	69 83 84	(est)
Mx 	169.22746
T	76 79 82 69 77	(lorem)
Mx 	196.70654
T	73 80 83 85 77	(ipsum)
Mx 	225.03523
T	68 79 76 79 82	(dolor)
Mx 	250.09244
T	83 73 84	(sit)
Mx 	138.44410
My 	177.33214
T	65 77 69 84	(amet)
Mx 	161.03622
T	76 79 82 69 77	(lorem)
Mx 	187.29533
T	73 80 83 85 77	(ipsum)
Mx 	214.40406
T	68 79 76 79 82	(dolor)
Mx 	238.24130
T	83 73 84	(sit)
Mx 	250.16447
T	65 77 69 84	(amet)
Mx 	138.44410
My 	191.33214
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	184.51278
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	229.41936
T	69 76 73 84 82	(elitr)
Mx 	248.76929
T	83 69 68	(sed)
Mx 	138.44410
My 	205.33214
T	68 73 65 77	(diam)
[1] Mx 	207.41759
My 	553.73265
T	17	(1)
New page
Mx 	40.97638
My 	37.33214
T	20	(4)
Mx 	47.86502
T	76 79 82 69 77	(lorem)
Mx 	74.41187
T	73 80 83 85 77	(ipsum)
Mx 	101.80832
T	68 79 76 79 82	(dolor)
Mx 	125.93329
T	83 73 84	(sit)
Mx 	138.14420
T	65 77 69 84	(amet)
Mx 	161.02405
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	207.38047
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	252.57477
T	69 76 73 84 82	(elitr)
Mx 	272.21244
T	83 69 68	(sed)
Mx 	288.09034
T	68 73 65 77	(diam)
Mx 	310.81882
T	78 79 78 85 77 89	(nonumy)
Mx 	347.34125
T	69 73 82 77 79 68	(eirmod)
Mx 	379.08829
T	84 69 77	(tem)
Mx 	395.18204
T	13	(-)
Mx 	20.97638
My 	51.33214
T	80 79 82	(por)
Mx 	38.08280
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	76.22926
T	85 84	(ut)
Mx 	87.78881
T	76 65 66 79 82 69	(labore)
Mx 	116.59445
T	69 84	(et)
Mx 	127.47528
T	68 79 76 79 82 69	(dolore)
Mx 	156.84244
T	77 65 71 78 65	(magna)
Mx 	187.26918
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	228.10118
T	69 82 65 84	(erat)
Mx 	247.53182
T	83 69 68	(sed)
Mx 	264.03278
T	68 73 65 77	(diam)
Mx 	287.38431
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	326.80031
T	65 84	(at)
Mx 	337.65184
T	86 69 82 79	(vero)
Mx 	359.01119
T	69 79 83	(eos)
Mx 	375.34125
T	69 84	(et)
Mx 	386.22208
T	65 67	(ac)
Mx 	395.18204
T	13	(-)
Mx 	20.97638
My 	65.33214
T	67 85	(cu)
Mx 	30.64435
T	83 65 77	(sam)
Mx 	50.28475
T	69 84	(et)
Mx 	61.50230
T	74 85 83 84 79	(justo)
Mx 	84.87806
T	68 85 79	(duo)
Mx 	103.56143
T	68 79 76 79 82 69 83	(dolores)
Mx 	137.12762
T	69 84	(et)
Mx 	148.34517
T	69 65	(ea)
Mx 	160.71018
T	82 69 66 85 77	(rebum)
Mx 	190.85351
T	83 84 69 84	(stet)
Mx 	209.37575
T	67 76 73 84 65	(clita)
Mx 	230.35405
T	75 65 83 68	(kasd)
Mx 	252.16242
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	298.12607
T	78 79	(no)
Mx 	311.82897
T	83 69 65	(sea)
Mx 	328.05629
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	367.60880
T	83 65 78 67 84 85 83	(sanctus)
Mx 	20.97638
My 	79.33214
T	69 83 84	(est)
Mx 	35.97265
T	76 79 82 69 77	(lorem)
Mx 	63.39569
T	73 80 83 85 77	(ipsum)
Mx 	91.66833
T	68 79 76 79 82	(dolor)
Mx 	116.66949
T	83 73 84	(sit)
Mx 	129.75658
T	65 77 69 84	(amet)
Mx 	153.51262
T	76 79 82 69 77	(lorem)
Mx 	180.93566
T	73 80 83 85 77	(ipsum)
Mx 	209.20830
T	68 79 76 79 82	(dolor)
Mx 	234.20946
T	83 73 84	(sit)
Mx 	247.29655
T	65 77 69 84	(amet)
Mx 	271.05259
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	318.28520
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	364.35569
T	69 76 73 84 82	(elitr)
Mx 	384.86954
T	83 69 68	(sed)
Mx 	20.97638
My 	93.33214
T	68 73 65 77	(diam)
[2] Mx 	207.41759
My 	553.68382
T	18	(2)
End page
Finish

